### PR TITLE
Use compare_stats_and_scores to compare scores

### PR DESCRIPTION
**Pull request by conorsloan**
The code was using compare_pokemon_stats which no longer exists.

===

Notes for @katieduplessis :
* This is a "pull request", a set of code changes for you to review and then approve or request changes
* The changes are a series of "commits" which are patches to the code that add, remove or change lines 
* Under the "files changed" tab you can see the "diff" which shows which lines were added, removed or changed. 

To add this code, I changed it in my editor and then ran:

``` bash
# Create a new branch called "fix-comparison" and check it out (make it my active branch on my Mac)
git checkout -b fix-comparison

# Commit the change, as in commit all the changes I've made to the record with an note of what I changed
# (Opens up a text window for me to type the commit message)
git commit -a

# Push my new branch to the "origin" remote so that GitHub knows about it
git push --set-upstream origin fix-comparison
```

then I went to GitHub to create a pull request. 

If you approve then this "commit" gets added to the main branch, i.e. "main" and you can use `git pull` to update the code on your computer with my changes.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -106,8 +106,7 @@ def run_round(player_chosen_username, opponent_given_username):
     )
     print("Which means....")
```
Comment (conorsloan) [inline main.py:107]: You can click on a line like this to comment on a particular line too

Comment (katieduplessis) [reply main.py:107]: Hey, look... A comment!

**Review by katieduplessis — APPROVED** [review]
@conorsloan Thanks for finding this bug! Have you checked out the opponent_random_stat fork? https://github.com/katieduplessis/Pokemon-Top-Trumps/blob/opponent_random_stat/main.py

**Comment by conorsloan** [comment]
Cool! I'm going to merge this into "main" now. I'll check out the other branch 👁️👄👁️

**Pull request merged**